source/_integrations/tod.markdown

```diff
@@ -6,20 +6,34 @@ ha_category:
 ha_release: 0.89
 ha_iot_class: Local Push
 ha_quality_scale: internal
+ha_config_flow: true
 ha_domain: tod
 ha_platforms:
   - binary_sensor
 ---
 
-The `tod` platform supports binary sensors which get their values by checking if the current time is within defined time ranges.
+The Times of the Day integration provides a binary sensor which get their values by checking if the current time is within defined time ranges.
+
+{% include integrations/config_flow.md %}
+{% configuration_basic %}
+Name:
+  description: The name the binary sensor should have. You can change it again later.
```

Comment:
```suggestion
  description: The name the binary sensor should have. This can be changed later.
```

Should we also mention that on and off-times can be changed?